README.md

```diff
@@ -1,3 +1,7 @@
+# Consultant Feedback Form
+
+[![Build Status](https://travis-ci.org/BuildIt-Poland/feedback-survey.svg?branch=master)](https://travis-ci.org/BuildIt-Poland/feedback-survey)
+
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 ## Available Scripts
```

Comment:
I would add some background story around this project. For now I'm only getting only info about scripts and what what was used to make this happen.

Reply:
it has also strong dependency on `backend` so I would mention it as well - to get full overview.